iac-common/src/main/java/org/sonar/iac/common/yaml/TreePredicates.java

```diff
@@ -39,6 +40,10 @@ public static Predicate<YamlTree> isEqualTo(String parameter) {
     return t -> TextUtils.isValue(t, parameter).isTrue();
   }
 
+  public static Predicate<YamlTree> isEqualToAnyOf(Collection<String> parameters) {
+    return t -> TextUtils.matchesValue(t, parameters::contains).isTrue();
+  }
```

Comment:
When creating new util methods you have to think about other usecase. IMO for now this method should be located in the check, as it's not used somewhere else.